### ISSUE TITLE
[Data] Missing training datasets for ALOHA simulation tasks + Paper-mentioned task mismatch

### ISSUE THREAD

**Issue by JimmyPang02** [long-context-dp/ldp#3]
Hi! I'm working with the ALOHA simulation environments in this repository and noticed that training data is only provided for one specific task, while the codebase supports many other ALOHA tasks. Additionally, there seems to be a mismatch between the paper-described task and the available dataset.

### Current Situation

**✅ Available:**
- Dataset: `data/aloha_twomodes_single/demos.hdf5`
- Task: `sim_singlearm_pickandplace_twomodes_scripted`
- Source: Google Drive link in README

**❌ Missing training data for:**
- `sim_transfer_cube`
- `sim_history_transfer_cube`
- `sim_pickandplace_samepos_scripted`
- `sim_pickandplace_origin_scripted`
- `sim_insertion`
- `sim_history_pickandplace_transfer_cube_scripted`
- `sim_singlearm_pickandplace_origin_scripted`

### Paper-Mentioned Task Mismatch

According to the paper, there's a "**long-horizon aloha, where one arm must pick up a block, move it to the center of the field of view, and return it precisely to its original location.**"

However:

**Current available task (`sim_singlearm_pickandplace_twomodes_scripted`):**
- Picks up block from random initial position
- Moves to one of **two predetermined target positions** (chosen based on proximity)
- Does **NOT** return to original location

**Paper-described task seems to match (`sim_pickandplace_samepos_scripted`):**
- Picks up block from random initial position  
- Must move it away from original location (distance > 15cm for reward)
- **Returns precisely to original location** (distance < 5cm for full reward)
- But **no training data available** for this task

### Questions

1. **Are training datasets available for the other ALOHA tasks?** If so, could you provide download links or instructions?

2. **Is there a specific dataset for the paper-mentioned "return to original location" task?** The available `aloha_twomodes_single` dataset seems to be for a different task variant.

3. **Task configuration clarification:** Should `sim_pickandplace_samepos_scripted` be the canonical "long-horizon ALOHA" task mentioned in the paper? If so, where can we find its training data?

4. **Are these missing tasks intended for evaluation only?** Should we generate our own training data through expert demonstrations?

### Code Evidence

In the config files, I see references to different tasks but all using the same dataset:
```yaml
# experiment_configs/aloha/transformer_aloha_noptp_nohist.yaml
task_name: sim_singlearm_pickandplace_twomodes_scripted
dataset_path: data/aloha_twomodes_single/demos.hdf5
```

### Expected Behavior

Either:
- Provide training datasets for all implemented ALOHA tasks, especially the paper-mentioned "return to original location" task
- Clear documentation indicating which tasks have training data vs. evaluation-only
- Instructions for collecting training data for missing tasks
- Clarification on which task corresponds to the paper description

### Environment
- Repository version: [latest]
- Following the installation instructions from README

Thank you for this great work! Looking forward to your guidance on reproducing the paper results.

---

**Comment by Zeffiretti** [comment]
Hi. I saw your issue and I’m running into the exact same problem. Do you know if there’s any workaround or fix yet?

**Comment by HaoranZhangumich** [comment]
yes, same issue, still did not see the dataset for reproducing the experiment
